experimental/generation/generator/packages/library/templates/standard/percentage.schema

```diff
@@ -1,14 +0,0 @@
-{
-    "$schema": "http://json-schema.org/draft-07/schema",
-    "type": "string",
-    "$entities": [
-        "percentage"
-    ],
-    "$templates": [
-        "string",
-        "percentageEntity.lu",
-        "percentageEntity.lg",
-        "percentageEntity-add-string.dialog",
```

Comment:
Why did you delete this file?  We need this so that someone can $ref a percentage.  The change was to make the type a number.